cmd/piper.go

```diff
@@ -78,17 +80,25 @@ func PrepareConfig(cmd *cobra.Command, metadata *config.StepData, stepName strin
 		stepConfig = config.GetStepConfigWithJSON(flagValues, GeneralConfig.StepConfigJSON, filters)
 	} else {
 		// use config & defaults
-
+		var customConfig io.ReadCloser
+		var err error
 		//accept that config file and defaults cannot be loaded since both are not mandatory here
-		customConfig, _ := openFile(GeneralConfig.CustomConfig)
+		if piperutils.FileExists(GeneralConfig.CustomConfig) {
+			if customConfig, err = openFile(GeneralConfig.CustomConfig); err != nil {
+				errors.Wrapf(err, "Cannot read '%s'", GeneralConfig.CustomConfig)
+			}
+		} else {
+			log.Entry().Debugf("Project config file '%s' does not exist. No project configuration available.", GeneralConfig.CustomConfig)
```

Comment:
minor: maybe switch to Info, then people will always see it in the log by default.

Reply:
done